templates/web/common/site_layout.tt.html

```diff
@@ -418,19 +418,20 @@ <h3 class="title-5 text-medium">[% lang('footer_discover_the_project') %]</h3>
 <script src="[% static_subdomain %]/js/dist/stikelem.js"></script>
 <script src="[% static_subdomain %]/js/dist/scrollNav.js"></script>
 
-<script>
-\$(function() {
-[% initjs %]
+
 [% IF server_options_producers_platform %]
+[% initjs %]
+<script>
```

Comment:
This is not the same !

[% inijs %] contains javascript, it have to be inside <script> !